tests/default_vars.sh

```diff
@@ -1245,6 +1245,8 @@ export DO_SAT_ADJ=.false.
 export KTHERM=2
 export TFREEZE_OPTION=mushy
 
+export FSCAV_AERO='"*:0.3","so2:0.0","msa:0.0","dms:0.0","nh3:0.4","nh4:0.6","bc1:0.6","bc2:0.6","dust1:0.6","dust2:0.6","dust3:0.6","dust4:0.6","dust5:0.6","seas1:0.5","seas2:0.5","seas3:0.5","seas4:0.5","seas5:0.5"'
```

Comment:
Since we defined the default value of "FSCAV_AERO" in default_vars.sh, do we still need to add this variable in each of the cpld test below?

Reply:
@junwang-noaa No we don't need to do it if it is defined in the default_vars.sh